meteor/client/ui/SegmentTimeline/SegmentTimeline.tsx

```diff
@@ -156,6 +156,10 @@ const SegmentTimelineZoom = class SegmentTimelineZoom extends React.Component<
 		return this.props.isLiveSegment ? this.calculateSegmentDuration() : this.state.totalSegmentDuration
 	}
 
+	calcTimeScale = (time: number) => {
```

Comment:
Perhaps this could be renamed to `convertTimeToPixels` or something like that?

Reply:
Yes thats a better name 👍

Reply:
Has been fixed :)